.github/workflows/development_infrastructure.yml

```diff
@@ -1315,7 +1315,8 @@ jobs:
                 WEBSITE_VNET_ROUTE_ALL=1 \
                 PORT=5000 \
                 WEBSITES_PORT=5000 \
-                RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD }}
+                RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD }} \
```

Comment:
I've updated the CLI scripts for the clearest variables, but I'm not 100% sure they are perfectly up to date.